### DISCUSSION TITLE
On-disk cache?

### DISCUSSION THREAD

**Discussion by hawkrives** [frog-pond/ccc-server#567] [Ideas]
Should we store our cached data in sqlite, instead of just in-memory?

Related: Pros/cons to defining a standard schema for storing the different data types, along with a consistent ID to be stored in the database cache? To allow us to get+serve historical data?

**Comment by drewvolz** [comment]
That's a good idea. Also +1 for historical data!

Also related: Potential to simplify the amount of requests a client makes in the future if the data we store is relational? Clients store data in sqlite? Transition to [more-data-on-homescreen for v3](https://github.com/StoDevX/AAO-React-Native/wiki/v3-Master-Plan#version-3) requiring single hydrating network request?

**Comment by drewvolz** [comment]
@hawkrives do you want to add your thoughts here for Caddy caching?
